docs/griptape-framework/structures/image-generation-drivers.md

```diff
@@ -0,0 +1,178 @@
+## Overview
+
+Image generation drivers are used by [image generation engines](../data/image-generation-engines.md) to build and execute API calls to image generation models.
+
+Use a driver to build an engine, then pass it to a tool for use by an [Agent](../structures/agents.md):
+
+```python
+from griptape.structures import Agent
+from griptape.engines import PromptImageGenerationEngine
+from griptape.drivers import OpenAiDalleImageGenerationDriver
+from griptape.tools import PromptImageGenerationClient, FileManager
+
+driver = OpenAiDalleImageGenerationDriver(
+    model="dall-e-3",
```

Comment:
Open Q: since Dall-E 3 requires a separate monthly subscription, would it be more accessible to start with Dall-E 2?

Reply:
These examples aren't prescriptive, but I updated this to `dall-e-2` because the Azure driver using our deployment requires `dall-e-3` and the downgrade here will save us a bit when running integration tests.